repo/packages/E/elastic/800/package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "packagingVersion": "4.0",
+  "upgradesFrom": [
+    "2.7.0-6.8.1"
```

Comment:
Similar question I asked in Kibana release PR, can customers upgrade from the previous version `6.6.1`? Can you try it? If it works, this would make the upgrade path for customers one step simpler and help us get them to upgrade faster.

Reply:
Answered  👇